sklearn/cross_validation.py

```diff
@@ -973,7 +973,7 @@ def _index_param_value(X, v, indices):
     return safe_indexing(v, indices)
 
 
-def cross_val_predict(estimator, X, y=None, cv=None, n_jobs=1,
+def cross_val_apply(estimator, X, y=None, cv=None, n_jobs=1, apply_func='predict',
```

Comment:
Please break this line so that it is at most 79 characters long, in accordance with PEP8